src/DefaultBuilder/src/WebHost.cs

```diff
@@ -206,6 +206,13 @@ public static IWebHostBuilder CreateDefaultBuilder(string[] args)
 
         internal static void ConfigureWebDefaults(IWebHostBuilder builder)
         {
+            builder.ConfigureAppConfiguration((ctx,cb) =>
+            {
+                if (ctx.HostingEnvironment.IsDevelopment())
+                {
+                    ctx.HostingEnvironment.UseStaticWebAssets();
```

Comment:
We automatically plug-in the static web assets during development, similar to user secrets

Reply:
Consider just making this not be an extension method. There's no reason it has to be, and it's not user-facing.

Reply:
Thinking about this more, I'm a little concerned about this being development only. Does this feature get used for files in nuget packages as well? If so, I don't think we can key this on `IsDevelopment()`.

Reply:
We totally can, I think explaining the different cases might clarify things.
* `dotnet run` builds the manifest and places it on the output folder. 
  * Under development (`dotnet run`) we will find the manifest and load the related assets.
  * If you want to do `dotnet run` and use the prod environment, you can simply call the extension method to enable it.
* `dotnet <<dll>>` (after publish) the files coming from packages and referenced projects will already be in their final locations, so it will work for development or production.
* Even if we were to unconditionally enable this, it will work based on the presence of the manifest (which we never publish) so it would work for dotnet run and not for dotnet publish.
* The reason to key it off development is to make it clear that this is a development feature.

Reply:
> Consider just making this not be an extension method. There's no reason it has to be, and it's not user-facing.

It was leftover from a previous implementation, didn't thought it matters given that it was internal, but I cleaned it up.

Reply:
> Even if we were to unconditionally enable this, it will work based on the presence of the manifest (which we never publish) so it would work for dotnet run and not for dotnet publish.

Then `IsDevelopment` isn't actually needed if the feature behaves the same with or without it.

> The reason to key it off development is to make it clear that this is a development feature.

Insufficient. If there's not a beahavioral difference then don't check IsDevelopment.

Reply:
> Then `IsDevelopment` isn't actually needed if the feature behaves the same with or without it.

It prevents it from being used on published apps if you copy the manifest by mistake (msbuild things happen)